src/utils.js

```diff
@@ -88,25 +85,28 @@ const getTextButton = (userId, eventDetail) => {
   return textButton;
 };
 
-const handleButtonEvent = (
-  e,
-  textButton,
-  userId,
-  eventDetail,
-  setGoToEditEvent,
-  setEventToEdit,
-  setTextButton
-) => {
-  const textEventButton = textButton;
-
-  if (textEventButton === "edit") {
+const handleButtonEvent = async (parameters) => {
+  const {
+    e,
+    textButton,
+    userId,
+    eventDetail,
+    setGoToEditEvent,
+    setEventToEdit,
+    setRefreshEvents,
+  } = parameters;
+
+  if (textButton === "edit") {
     goToEditEventFunction(e, eventDetail, setGoToEditEvent, setEventToEdit);
```

Comment:
we need to organice the logic inside `goToEditEventFunction` we dont need to pass e, we already have the eventId, so inside that function we only should have this:
```
setGoToEditEvent(true);
setEventToEdit(eventId);
```